docs/install/local-docker.md

```diff
@@ -0,0 +1,83 @@
+# Install MLRun locally using Docker
+
+You can install and use MLRun and Nuclio locally on your computer, this will not include all the services and elastic 
+scaling capabilities which you can get with the Kubernetes based deployment but is much simpler to start with.
+
+```{admonition} Note
+- Using Docker is limited to local, Nuclio, and Serving runtimes and local pipelines.
+```
+
+The installation is done using [`docker compose`](https://docs.docker.com/compose/) and will deploy MLRun service, 
+MLRun UI, Nuclio serverless engine, and optionally Jupyter server. 
+
+There are two installation options:
+- [**Use MLRun with your own client (PyCharm, VSCode, Jupyter)**](#use-mlrun-with-your-own-client)
+- [**Use MLRun with MLRun Jupyter image (pre loaded with examples/demos)**](#use-mlrun-with-mlrun-jupyter-image)
+
+In both cased you need to set the `SHARED_DIR` environment variable to point to a host path for storing MLRun artifacts and DB, 
+for example `export SHARED_DIR=~/mlrun-data` (or use `set SHARED_DIR=c:\mlrun-data` in windows), make sure the directory exist.
+
+It is recommended to set the `HOST_IP` variable with your computer IP address (required for Nuclio dashboard), 
+you can check your IP address using the `ip addr` command (or `ipconfig` in windows).
```

Comment:
on mac you dont always have `ip` installed, unless you `brew install` its package (dont remember its name, can google it).
you can use `ifconfig en0| grep "inet[ ]" | awk '{print $2}'` to determine that.

for niceness, if you *do* have `ip` command, you can get your external ip address by
`ip route get 8.8.8.8 | awk -F"src " 'NR==1{split($2,a," ");print a[1]}'`

which yields your external ip address

Reply:
i will add `ifconfig` rather not put scripts since a. may not work (e.g. no en0), too specific (may have few addresses and its actually preferred not to take the dynamic/dhcp one), i trust they will know how to find their IP